share/functions/__fish_print_ninja_targets.fish

```diff
@@ -0,0 +1,5 @@
+function __fish_print_ninja_targets
+    if [ -f build.ninja ]
+        ninja -t targets | sed 's/:.*//'
```

Comment:
You mean something like  

    ...
    ninja -t targets | string replace -r ':.*' '' 2> /dev/null

?